### ISSUE TITLE
bcharts.com.br

### ISSUE THREAD

**Issue by DevWannabe-dot** [qsniyg/maxurl#988]
original: https://bcharts.com.br/user_avatar/bcharts.com.br/hades/70/17261_2.png
biggest I found: https://bcharts.com.br/user_avatar/bcharts.com.br/hades/560/17261_2.png

the site is really slow and very few people use it, but would be useful for me and surely others. Thanks!

**Comment by qsniyg** [comment]
I believe the maximum size is 1000, but it upscales. I can't find a way to get non-upscaled images from there. I could add a "possibly upscaled images" rule for it though, if you wish.

**Comment by DevWannabe-dot** [comment]
Yeah, that would be great! Miles better than not having a rule for it at all, and probably there isn't even a logical original image size, but after a certain size it only upscales, which can't be guessed for all images. 
The error image is something like: https://bcharts.com.br/user_avatar/bcharts.com.br/hades/10000/17261_2.png

Thanks!

**Comment by DevWannabe-dot** [comment]
@qsniyg oops
